downloadCOAMPSTC.sh

```diff
@@ -0,0 +1,490 @@
+#!/bin/bash
```

Comment:
We need to start putting all scripts in `./bin` or somewhere else that's in `PATH`. If it's not in `PATH`, then we need to add it there. `$SCRIPTDIR` is not currently in `PATH` but probably should be just for the existing scripts there.

Reply:
can do